crates/graphql-federated-graph/src/federated_graph.rs

```diff
@@ -58,42 +58,22 @@ pub struct FederatedGraph {
 
 impl FederatedGraph {
     #[cfg(all(feature = "from_sdl", feature = "extension"))]
-    pub async fn from_sdl_with_extensions(sdl: &str) -> Result<Self, crate::DomainError> {
+    pub async fn from_sdl(sdl: &str) -> Result<Self, crate::DomainError> {
```

Comment:
We'll have to land back on an extensions free API for the next release of the crate, I think. But based on what we discussed this morning, the async part will move to engine::Schema building, right?

Reply:
yeah removing that stuff and just imitating the `join__Graph` handling in a PR right now